src/CMakeLists.txt

```diff
@@ -63,31 +61,37 @@ endif()
 target_precompile_headers(CemuBin PRIVATE Common/precompiled.h)
 
 if(WIN32)
-  target_sources(CemuBin PRIVATE
-  resource/cemu.rc
-  )
+	target_sources(CemuBin PRIVATE
+	resource/cemu.rc
+)
 endif()
 
 set_property(TARGET CemuBin PROPERTY MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
 
 set_target_properties(CemuBin PROPERTIES
-        RUNTIME_OUTPUT_DIRECTORY_DEBUG ${CMAKE_CURRENT_SOURCE_DIR}/../bin/
-        RUNTIME_OUTPUT_DIRECTORY_RELEASE ${CMAKE_CURRENT_SOURCE_DIR}/../bin/
-        OUTPUT_NAME "Cemu"
-        )
+	RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../bin/
+	OUTPUT_NAME "Cemu"
+)
+
+target_link_libraries(CemuBin PRIVATE
+	CemuAudio
+	CemuCafe
+	CemuCommon
+	CemuComponents
+	CemuConfig
+	CemuGui
+	CemuInput
+	CemuUtil
+)
 
-target_link_libraries(CemuBin PRIVATE CemuCommon CemuComponents CemuCafe CemuConfig CemuGui CemuAudio CemuInput CemuUtil)
 target_link_libraries(CemuBin PRIVATE CemuAsm)
-target_link_libraries(CemuBin PRIVATE OpenSSL::SSL)
+target_link_libraries(CemuBin PRIVATE OpenSSL::SSL OpenSSL::Crypto)
 target_link_libraries(CemuBin PRIVATE ZLIB::ZLIB)
-target_link_libraries(CemuBin PRIVATE ${wxWidgets_LIBRARIES})
 target_link_libraries(CemuBin PRIVATE CURL::libcurl)
-target_link_libraries(CemuBin PRIVATE imgui::imgui)
 target_link_libraries(CemuBin PRIVATE pugixml pugixml::static pugixml::pugixml)
```

Comment:
I have to remove `pugixml::static` on Gentoo to get past this line.

Reply:
Yeah I still have to handle the pigixml dependency